code/src/css/PopularList.css

```diff
@@ -0,0 +1,51 @@
+.list{
+    display:flex;
+    flex-wrap: wrap;
+    width: 100vw;
+  }
+  
+  .poster {
+    width: 25vw;
+  }
+  
+  .movieCard {
+    position: relative;
+  }
+  
+  .details {
+    position: absolute;
+    bottom: 1%;
+    color: #fff;
+    padding: 5%;
+  }
+
+  h1 {
+    font-size: 32px;
+  }
```

Comment:
To make the overlay-text pop and be easier to read you could increase the font-size of the title.